custom-login/README.md

```diff
@@ -52,13 +52,20 @@ Now start the app server:
 ng serve
 ```
 
+> **Note:** If you're on a windows machine, you might get an error like this
+```
+ng serve contains error Unknown browser query `basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")`
```

Comment:
Using the callout `>` and a code block makes this render a little funny.

Maybe move the code block into `back-ticks` so it fits in one section?

> blah blah
> blah
> `error goes here`
> **Note**: Big scary warning